v1/plugin/plugin.go

```diff
@@ -52,12 +52,11 @@ var (
 		version int
 		opts    []MetaOpt
 	}
-)
-
-func init() {
-	app = cli.NewApp()
-	app.Flags = []cli.Flag{
+	// Flags required by the plugin lib flags - plugin authors can provide their
+	// own flags.
```

Comment:
It would be great to point here to snap-plugin-collector-rand/rand/rand.go as a place where the example of usage can be found.